plugins/terraform/src/components/TerraformLatestRunContent/utils.ts

```diff
@@ -0,0 +1,10 @@
+const colorMap: Record<string, string> = {
```

Comment:
This looks like the same utils file in `DenseTable`. Can this be put in the `src/utils` dir and then used from there in both places?

Reply:
This module is not actually referenced and was committed in error.

Only the DenseTable utils version is being used.

Will remove.